Some refactor and adding log interpretation. (#14)
refractionPOINT/lce_doc: docs/troubleshooting.md

disconnecting from the cloud for a few seconds. This means that if you temporarily see a sensor as offline when you
expect it to be online, give it 30 seconds, in most situations it will come back online with 5 seconds.

## Sensor not Connecting
Sensors connect to the LimaCharlie.io cloud via an SSL connection on port 443. Make sure your network allows such
a connection. It is a very common port usually used for HTTPS so an issue is highly unlikely.

> The sensor uses a pinned SSL certificate to talk to the cloud. This means that if you are in a network that enforces SSL inspection (a man-in-the-middle of the SSL connections sometimes used in large corporate environments), this may prevent the sensor from connecting. LimaCharlie uses a pinned certificate to ensure the highest level of security possible as usage of off-the-shelf certificates can be leveraged by state sponsored (or advanced) attackers.

Reviewer: ```suggestion
The sensor uses a pinned SSL certificate to talk to the cloud. This means that if you are in a network that enforces SSL inspection (a man-in-the-middle of the SSL connections sometimes used in large corporate environments), this may prevent the sensor from connecting. LimaCharlie uses a pinned certificate to ensure the highest level of security possible as usage of off-the-shelf certificates can be leveraged by state-sponsored (or advanced) attackers.
```

ssl connected
headers sent
channel up
```

> If you are having trouble having your sensor connect to the cloud, we recommend:

Reviewer: ```suggestion
If you are having trouble getting your sensor connected to the cloud, we recommend:
```

> 1. Restart the LimaCharlie service.

Reviewer: ```suggestion
1. Restart the LimaCharlie service on the host.
```
